storage/tokudb/ha_tokudb.cc

```diff
@@ -8076,6 +8079,7 @@ bool ha_tokudb::rpl_lookup_rows() {
 #include "ha_tokudb_mrr_mysql.cc"
 
 // handlerton
+#include "ha_tokupart.cc"
```

Comment:
Maybe let's stop doing this?

Reply:
I tried to fix it, but it's not enough just to remove the corresponding "include" from the .cc file and add ha_tokupart.cc to the correspondent list in cmake-file. "Include" dependencies are also must be satisfied. And when some files from tokudb directory are included into ha_tokupart.cc file, the files itself must be fixed. And there are no one or two fixes, there are several tens fixes, and that's is why I think there must be separate task for this.

Reply:
Yes, this is unfortunately just the way it is.  I tried at one point to pull these apart and it was a holy mess.